src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -193,6 +198,78 @@ class SpanDetail extends React.Component<Props, State> {
     );
   }
 
+  renderSpanErrorMessage() {
+    const {orgId, spanErrors, totalNumberOfErrors, span, trace, eventView} = this.props;
+
+    if (spanErrors.length === 0 || totalNumberOfErrors === 0 || isGapSpan(span)) {
+      return null;
+    }
+
+    // invariant: spanErrors.length <= totalNumberOfErrors
+
+    const eventSlug = generateEventSlug(spanErrors[0]);
+
+    const {start, end} = getTraceDateTimeRange({
+      start: trace.traceStartTimestamp,
+      end: trace.traceEndTimestamp,
+    });
+
+    const errorsEventView = EventView.fromSavedQuery({
+      id: undefined,
+      name: `Error events associated with span ${span.span_id}`,
+      fields: ['title', 'project', 'issue', 'timestamp'],
+      orderby: '-timestamp',
+      query: `event.type:error trace:${span.trace_id} trace.span:${span.span_id}`,
+      projects: eventView.project,
+      version: 2,
+      start,
+      end,
+    });
+
+    const target =
+      spanErrors.length === 1
+        ? {
+            pathname: eventDetailsRoute({
+              orgSlug: orgId,
+              eventSlug,
+            }),
+          }
+        : errorsEventView.getResultsViewUrlTarget(orgId);
+
+    const message =
+      totalNumberOfErrors === 1 ? (
+        <Link to={target}>
+          <span>{t('An error event occurred in this span.')}</span>
+        </Link>
+      ) : spanErrors.length === totalNumberOfErrors ? (
+        <div>
+          <Link to={target}>
+            <span>{`${totalNumberOfErrors} error events`}</span>
+          </Link>
+          <span>{' occurred in this span.'}</span>
```

Comment:
```suggestion
          <span>{t(' occurred in this span.')}</span>
```
This would ideally all be one translated phrase as well using tct() as right now it cannot be translated.

Reply:
https://github.com/getsentry/sentry/pull/18025/commits/c68ef3cb460aea1004255fae94dd107f32dd3d7b